website_and_docs/content/documentation/webdriver/interactions/windows.en.md

```diff
@@ -33,40 +35,15 @@ Clicking a link which opens in a
 <a href="https://seleniumhq.github.io" target="_blank"> new window</a>
 will focus the new window or tab on screen, but WebDriver will not know which
 window the Operating System considers active.  To work with the new window
-you will need to switch to it. If you have only two tabs or windows open,
-and you know which window you start with, by the process of elimination
-you can loop over both windows or tabs that WebDriver can see, and switch
-to the one which is not the original.
-
-However, Selenium 4 provides a new api [NewWindow](#create-new-window-or-new-tab-and-switch) 
```

Comment:
Hi @pallavigitwork ,
Any reason why this was removed ?

Reply:
@harsha509 it was removed because we are mentioning it where we discuss about new window and new tab. It made more sense and clarity in that section. I felt here was creating some confusion so that is why. The information is still very much on this page.